utils/build-script

```diff
@@ -774,6 +778,115 @@ class BuildScriptInvocation(object):
 
         return options
 
+    def compute_product_classes(self):
```

Comment:
We will eventually need this to be host-specific, so it doesn't make sense to be a property.